tests/utils/autoTester.js

```diff
@@ -334,6 +362,9 @@ module.exports.runTest = function (testType) {
         return;
     }
 
+    // Setup the screen, camera, snapshots for the next test  
+    setUpTest();
```

Comment:
THis call is happening just before the first step lambda but INSIDE the test case Func AFTER the initial block of instructions.
So this not the same as what it was before.
More importantly, we said we wanted that setup  that defines the Origin Frame and the Validation Camera should happen before ANY thing running in the test case lambda so you could rely on that.

So i still think that what we want is a simple function "runOneTestCase( theTestCase , testType )"
Where you call
 #1 setupTest(theTestCase)
#2 theTestCase Func(testType)